src/features/auth/Register.tsx

```diff
@@ -1,21 +1,38 @@
-import Box from "@mui/material/Box";
-import Typography from "@mui/material/Typography";
 import FormControl from "@mui/material/FormControl";
 import RegisterForm from "./register-form/RegisterForm";
 import { FunctionComponent, useState, useCallback } from "react";
 import registerService from "features/auth/services/register.service";
 
 import useRedirectWhenRegistered from "./use-redirect-when-registered";
+import Div100vh from 'react-div-100vh'
 
 const Register: FunctionComponent = () => {
   const { loading, onSubmit } = useRegister();
+
+
+  const mediaQueryStyles = {
+    '@media (minWidth: 600px)': {
+      minHeight: '500px',
+    },
+    '@media (minWidth: 900px)': {
+      minHeight: '600px',
+    },
+  };
+
```

Comment:
```suggestion

  const mediaQueryStyles = {
    '@media (minWidth: 600px)': {
      minHeight: '500px',
    },
    '@media (minWidth: 900px)': {
      minHeight: '600px',
    },
  };
 
  const registerProfileStyles = {
        display: 'flex',
      flexDirection: 'column',
      backgroundColor: 'blue',
      alignItems: 'center',
      justifyContent: 'center',
      minHeight: '800px',
      minWidth: '300px',
      ...mediaQueryStyles,
  };
```
Esto deberia ir en otro archivo de estilos